paper/paper.md

```diff
@@ -21,158 +21,63 @@ authors:
+The faster they get noticed, the faster analysis scripts, dashboards and data pipelines can be updated.
 
 Without dedicated systems to detect them, changes can often go unnoticed for months.
 `datapact` enables such a system to be built easily, so changes are noticed as the data comes in.
-It is a Python library for expressing + verifying characteristics on DataFrames, and can be embedded right within the Python scripts and Jupyter notebooks that are already being used for analysis.
-Different tests are needed, both simple sanity checks ("the ICU patient's age should be between 0 and 150") and more complex characteristics ("the age should be normal distributed").
-Specialized tests should be easily addable via custom code.
+It is a Python library for expressing and verifying characteristics on DataFrames, and can be embedded right within the Python scripts and Jupyter notebooks that are already being used for analysis.
+Different tests are needed, both simple sanity checks ("the patient's age should be between 0 and 125") and more complex characteristics ("the age should be normally distributed").
+More specialized tests can easily be added via custom code.
 
-Great Expectations (@Gong_Great_Expectations) is an existing framework that addresses these issues via Pipeline Tests. It can be hard to learn and hard to set up, especially for non-engineering-heavy organisations.
+Existing projects addressing this need (e.g. Great Expectations (@Gong_Great_Expectations)) come with comparatively high entrance hurdles that are hard to overcome, particularly for non engineering heavy organisations.
 
-`datapact` is designed to be used by researchers, industry data scientists and data engineers alike.
-For teams there's `Datapact Track`: the optional web-app that tracks test results, and notifies about failures via E-Mail, Slack, MicroSoft Teams or PagerDuty.
+With `datapact`, we provide an easily usable library that is designed to be used by researchers, industry data scientists and data engineers alike.
+For teams there is `Datapact Track`: the optional web-app that tracks test results, and notifies about failures via E-Mail, Slack, Microsoft Teams, or PagerDuty.
 
 # Method
 
-The pain points around continous data verification were identified by the author during a research project involving continously surveyed mobility data.
-These were validated via qualitative interviews across a variety of research and industry organisations.
+Qualitative interviews across a variety of research and industry organisations were conducted to learn about the problem of unnoticed dataset shift.
 After evaluating prior art, the idea of an assertion library was identified as a potential solution to the problem.
 To ensure that `datapact`'s developer-facing API is intuitive + ergonomic to use, the authors performed informal experiments where developers without prior knowledge about the problem were asked to explain what they think a given `datapact` snippet did.
```

Comment:
I would avoid using `+` since it is not a mathematical expression.
```suggestion
To ensure that `datapact`'s developer-facing API is intuitive and ergonomic to use, the authors performed informal experiments where developers without prior knowledge about the problem were asked to explain what they think a given `datapact` snippet did.
```